projects/vdk-control-cli/tests/vdk/internal/control/command_groups/job/test_execute.py

```diff
@@ -1,7 +1,6 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 import json
-import os
 from unittest.mock import patch
 
```

Comment:
May you link if described in VEP-1739, what happens in case of local execution of a data job, with added `python_version` in config.ini - in case of mismatch with local python environment?

Reply:
It is not yet described in the proposal. Will update the vep in a separate PR.
As to the question of what happens if the python_version set in the config is different from the one of the local environment, the answer is, nothing. The `python_version` property (if set, as it is optional) is not taken into account until the deployment of the job is initiated.
Do you think we need some form of validation that the `python_version` is the same as the local environment?

Reply:
I think it may be very useful to alert the user in case of a mismatch between (cloud) python version targeted and local python env setup (used for development). This may prevent a lot of confusion.

Reply:
This is a good idea. I'll add this functionality in a separate PR, as the change would need to be made in vdk-core.
Thank you for the review